site-permissions/site-permissions-impl/src/main/java/com/duckduckgo/site/permissions/impl/SitePermissionsDialogActivityLauncher.kt

```diff
@@ -100,18 +102,24 @@ class SitePermissionsDialogActivityLauncher @Inject constructor(
         pixelParamValue: String
     ) {
         pixel.fire(SitePermissionsPixelName.SITE_PERMISSION_DIALOG_SHOWN, mapOf(PixelParameter.SITE_PERMISSION to pixelParamValue))
-        AlertDialog.Builder(activity).apply {
-            setTitle(String.format(activity.getString(titleRes), url.websiteFromGeoLocationsApiOrigin()))
-            setPositiveButton(R.string.sitePermissionsDialogAllowButton) { _, _ ->
-                pixel.fire(SitePermissionsPixelName.SITE_PERMISSION_DIALOG_ALLOWED, mapOf(PixelParameter.SITE_PERMISSION to pixelParamValue))
-                onPermissionAllowed()
-            }
-            setNegativeButton(R.string.sitePermissionsDialogDenyButton) { _, _ ->
-                pixel.fire(SitePermissionsPixelName.SITE_PERMISSION_DIALOG_DENIED, mapOf(PixelParameter.SITE_PERMISSION to pixelParamValue))
-                sitePermissionRequest.deny()
-            }
-            setCancelable(false)
-            show()
+        (activity as FragmentActivity?)?.supportFragmentManager?. let { fragmentManager ->
```

Comment:
code style: I think we should remove the space between the `.` and the `.let`
```suggestion
        (activity as FragmentActivity?)?.supportFragmentManager?.let { fragmentManager ->
```